docs/tutorials/porch-development-environment/bin/setup.sh

```diff
@@ -110,11 +120,111 @@ if ! git rev-parse -q --verify refs/remotes/origin/main >/dev/null; then
 else
   echo "main branch already exists in git repo."
 fi
+cd "${git_root}"
+rm -fr "$TMP_DIR"
 
 ############################################
-h1 "Clean up"
-cd "$self_dir"
-rm -fr "$TMP_DIR"
+h1 Generate certs and keys
+cd "${git_root}"
+deployments/local/makekeys.sh
+
+############################################
+h1 Install CRDs
+cd "${git_root}"
+kubectl apply -f api/porchconfig/v1alpha1/config.porch.kpt.dev_repositories.yaml
+kubectl apply -f api/porchconfig/v1alpha1/config.porch.kpt.dev_functions.yaml
+kubectl apply -f controllers/config/crd/bases/config.porch.kpt.dev_packagevariants.yaml
+kubectl apply -f controllers/config/crd/bases/config.porch.kpt.dev_packagevariantsets.yaml
+kubectl apply -f internal/api/porchinternal/v1alpha1/config.porch.kpt.dev_packagerevs.yaml
+
+
+############################################
+h1 Load container images into kind cluster
+cd "${git_root}"
+export IMAGE_TAG=v2.0.0
+export KIND_CONTEXT_NAME="$porch_cluster_name"
+if ! docker exec -it "$porch_cluster_name-control-plane" crictl images | grep -q docker.io/nephio/test-git-server ; then
+  make build-images 
+  kind load docker-image docker.io/nephio/porch-controllers:${IMAGE_TAG} -n ${KIND_CONTEXT_NAME}
+  kind load docker-image docker.io/nephio/porch-function-runner:${IMAGE_TAG} -n ${KIND_CONTEXT_NAME}
+  kind load docker-image docker.io/nephio/porch-wrapper-server:${IMAGE_TAG} -n ${KIND_CONTEXT_NAME}
+  kind load docker-image docker.io/nephio/test-git-server:${IMAGE_TAG} -n ${KIND_CONTEXT_NAME}
+else
+  echo "Images already loaded into kind cluster."
+fi
+
+############################################
+h1 Install all porch components, except porch-server
+cd "${git_root}"
+make deployment-config-no-sa
+cd .build/deploy-no-sa
+# expose function-runner to local processes
+kpt fn eval \
+  --image gcr.io/kpt-fn/starlark:v0.5.0 \
+  --match-kind Service \
+  --match-name function-runner \
+  --match-namespace porch-system \
+  -- "ip=${function_runner_ip}"  'source=
+ip = ctx.resource_list["functionConfig"]["data"]["ip"]
+for resource in ctx.resource_list["items"]:
+  resource["metadata"].setdefault("annotations", {})["metallb.universe.tf/loadBalancerIPs"] = ip
+  resource["spec"]["type"] = "LoadBalancer"
+  resource["spec"]["ports"][0]["nodePort"] = 30001'
+# "remove" porch-server from package
```

Comment:
Not sure how we "remove" the porch server here? Just by setting the annotation?

Reply:
yes, exactly.
Setting the annotation will make `kpt live apply` to ignore it. (Similarly to ConfigSync.)